theme/js/components/dataset/card-lg.vue

```diff
@@ -39,18 +39,20 @@
             <small v-if="acronym">{{ acronym }}</small>
           </a>
         </h4>
-        <span class="not-enlarged" v-if="organization || owner">
+        <p class="fr-m-0 fr-text--sm" v-if="organization || owner">
           {{ $t('From') }}
-          <a :href="organization.page" v-if="organization">
+          <span class="not-enlarged" v-if="organization">
+            <a :href="organization.page">
               <OrganizationNameWithCertificate :organization="organization" />
-          </a>
+            </a>
+          </span>
           <template v-if="owner">{{ownerName}}</template>
-        </span>
+        </p>
         <p class="fr-mt-1w fr-mb-2w fr-hidden fr-unhidden-sm">
           {{ $filters.excerpt(description, 160) }}
         </p>
-        <p class="fr-mb-0 text-mention-grey">
-          <Tooltip class="fr-hidden inline-sm">
+        <p class="fr-m-0 fr-grid-row fr-grid-row--middle fr-text--sm text-mention-grey">
+          <Tooltip class="fr-hidden fr-grid-row fr-grid-row--middle flex-sm">
```

Comment:
I think we can use a larger width to avoid this with French translations.

![Xnapper-2023-02-21-13 51 26](https://user-images.githubusercontent.com/8336712/220349559-a7836046-3081-494c-9d46-8be393a54e83.png)

What do you think ?

Reply:
Yep, would work for me :)